DFC.App.ExploreCareers.Data/Contracts/IApiDataModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace DFC.App.ExploreCareers.Data.Contracts
+{
+    public interface IApiDataModel
```

Comment:
You do not  need to define this interface yourself as it is defined in the content nuget